templates/shared/contextual_footers/_download_server_installation.html

```diff
@@ -3,6 +3,6 @@ <h3 class="p-heading--four">Installation guides</h3>
   <p>If you need some help installing Ubuntu, please check out our step-by-step guides.</p>
   <ul class="p-list">
       <li><a class="p-link--external" href="https://tutorials.ubuntu.com/tutorial/tutorial-install-ubuntu-server" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'server instructions', 'eventLabel' : 'Installation guides', 'eventValue' : undefined });">Installation instructions for Ubuntu Server</a></li><li>
-      </li><li><a href="/download/cloud/install-openstack-with-autopilot" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'autopilot instructions', 'eventLabel' : 'Installation guides', 'eventValue' : undefined });">Installation instructions for Ubuntu OpenStack&nbsp;&rsaquo;</a></li>
+      </li><li><a href="/download/cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'cloud instructions', 'eventLabel' : 'Installation guides', 'eventValue' : undefined });">Installation instructions for Ubuntu OpenStack&nbsp;&rsaquo;</a></li>
```

Comment:
Shouldn't this one link to the OpenStack page too?

Reply:
again, about installing